app/scheduled_workers/decidim_initiatives.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class DecidimInitiatives
+  include Sidekiq::Worker
+
+  def perform
+    system("bundle exec rake decidim_initiatives:check_validating")
+    system("bundle exec rake decidim_initiatives:check_published")
+    system("bundle exec rake decidim_initiatives:notify_progress")
```

Comment:
We would like to have one worker per rake task that way they are 100% independent in front of failure.

Reply:
Can you review it now, please?